dev/src/commands/AbstractGenerateStubCommand.ts

```diff
@@ -88,27 +97,37 @@ export default abstract class AbstractGenerateStubCommand extends AbstractGenera
                     case "go":
                     case "Go":
                         currentGeneratorTypes = this.goTypes;
-                        preferredSourceLocation = ".";
                         break;
                     case "java":
                     case "Java":
-                        currentGeneratorTypes = this.javaTypes;
-                        preferredSourceLocation = "src";
+                        if ("spring" === this.projectType) {
+                            currentGeneratorTypes = this.javaSpringTypes;
+                        } else if (this.projectType.toLowerCase().indexOf("liberty") >= 0 ||
+                                this.projectType.toLowerCase().indexOf("docker") >= 0) {
+                            currentGeneratorTypes = this.codewindTypes;
+                        } else if (this.projectType.toLowerCase().indexOf("appsody") > 0) { // appsodyExtension
```

Comment:
For an appsody project, the type projetctType is "appsodyExtension" but the indexOf should be >= 0

Reply:
@sghung , yes, it should be >=0
Yes, the extension should be appsodyExtension but I was just looking specifically for 'appsody' which should apply to this if-clause (with the corrected >= comparator)
See comment next to it